test/list.test.js

```diff
@@ -7,6 +7,10 @@
 
 const should = require('./init.js');
 const List = require('../lib/list');
+const parentRefHelper = require('./helpers/parentRef-helper');
```

Comment:
I find it difficult to understand in the context of this test what is `parentRefHelper` doing. Please use a more descriptive name, e.g.

```suggestion
const createTestSetupForParentRef = require('./helpers/parentRef-helper');
```

Reply:
renamed variable